src/QualificationRunner/Program.cs

```diff
@@ -36,26 +36,30 @@ static int Main(string[] args)
 
       private static void startCommand<TRunOptions>(CLICommand<TRunOptions> command)
       {
-         var logger = initializeLogger(command);
+         var (logger, loggerFactory) = initializeLogger(command);
 
          logger.AddInfo($"Starting {command.Name.ToLower()}");
          logger.AddDebug($"Arguments:\n{command}");
-         
+
          var runner = IoC.Resolve<IBatchRunner<TRunOptions>>();
-         try
-         {
-            runner.RunBatchAsync(command.ToRunOptions()).Wait();
-            logger.AddInfo($"{command.Name} finished");
-         }
-         catch (Exception e)
+
+         using (loggerFactory)

```

Comment:
disposing the loggerFactory before closing will help flush out any messages that were not shown to user apparently.

Can you try again?